pkg/scheduler/apis/config/types.go

```diff
@@ -85,6 +85,9 @@ type KubeSchedulerConfiguration struct {
 	// then scheduler stops finding further feasible nodes once it finds 150 feasible ones.
 	// When the value is 0, default percentage (5%--50% based on the size of the cluster) of the
 	// nodes will be scored.
+	// Note: It will be overridden by the profile-level score percentage when that is available.
+	// DEPRECATED: Please use scheduler profiles to configure this instead.
+	// TODO: remove this in v1beta2 (https://github.com/kubernetes/kubernetes/issues/95446)
```

Comment:
```TODO(#95446): Remove PercentageOfNodesToScore from KubeSchedulerConfiguration once v1beta1 is removed.```

Put it in it's own comment block above, so that it's not part of the documentation.